src/parser/signup.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function parseSignup(message) {
+module.exports = function parseSignup(message, discordId) {
```

Comment:
After adding the id, we should validate for it to not be undefined as well.

Reply:
Great catch, I will definitely look into this as soon as possible.

Reply:
Thanks, I will definitely look into this as soon as I can.